dotCMS/src/main/java/com/dotmarketing/util/ImportExportUtil.java

```diff
@@ -1312,6 +1314,19 @@ private void doXMLFileImport(File f, PrintWriter out, ObjectFilter filter)throws
                     dc.addParam(portletPreferences.getPreferences());
                     dc.getResults();
                 }
+            }else if (_importClass.equals(Tree.class)) {
+                for (int j = 0; j < l.size(); j++) {
+                	Tree tree = (Tree)l.get(j);
+                	try{
+	                	LocalTransaction.wrap(()->{
+	                		TreeFactory.insertTree(tree);
+	                	});
+                	}catch(Throwable r){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Catch Exception instead of Throwable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)